library/src/main/java/com/google/android/exoplayer/text/eia608/ClosedCaptionCtrl.java

```diff
@@ -69,6 +71,45 @@
   public static final byte TAB_OFFSET_CHAN_1 = 0x17;
   public static final byte TAB_OFFSET_CHAN_2 = 0x1F;
 
+  private static final String TAG = "Eia608CaptionControl";
+
+  private static final int[] COLOR_VALUES = {
+          Color.WHITE,
+          Color.GREEN,
+          Color.BLUE,
+          Color.CYAN,
+          Color.RED,
+          Color.YELLOW,
+          Color.MAGENTA,
+          Color.TRANSPARENT // Last color setting should be kept
+  };
+
+  // for debug purposes
```

Comment:
Can we remove all the debug stuff from committed code? Feel free to keep it locally, but it's probably >50% of the whole change to this file! Note that getIndentation is also not used anywhere, so that method can go or be deferred until a subsequent change also.